src/lib/shell-imp/index.ts

```diff
@@ -0,0 +1,104 @@
+import {IImpactModelInterface} from '../interfaces';
+import * as cp from 'child_process';
+import {KeyValuePair} from '../../types/boavizta';
+import * as yaml from 'js-yaml';
+
+export class ShellModel implements IImpactModelInterface {
+  // Defined for compatibility. Not used in TEADS.
```

Comment:
teads?

Reply:
It was copied from TEADS. I will remove it.